src/main/resources/init_db.sql

```diff
@@ -3,18 +3,40 @@ USE `taxi`;
 
 DROP TABLE IF EXISTS `manufacturers`;
 CREATE TABLE `manufacturers` (
-                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
-                                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+                                 `id` bigint NOT NULL AUTO_INCREMENT,
+                                 `name` varchar(255) DEFAULT NULL,
+                                 `country` varchar(255) DEFAULT NULL,
+                                 `is_deleted` tinyint DEFAULT '0',
+                                 PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=45 DEFAULT CHARSET=utf8mb3;
 
 DROP TABLE IF EXISTS `drivers`;
 CREATE TABLE `drivers` (
-                                  `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                  `name` VARCHAR(225) NOT NULL,
-                                  `license_number` VARCHAR(225) NOT NULL,
-                                  `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                  PRIMARY KEY (`id`),
-                                  UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
-                                  UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+                           `id` bigint NOT NULL AUTO_INCREMENT,
+                           `name` varchar(45) DEFAULT NULL,
+                           `licenseNumber` varchar(255) DEFAULT NULL,
+                           `is_deleted` tinyint(1) DEFAULT '0',
+                           `manufacturer_id` bigint NOT NULL,
+                           PRIMARY KEY (`id`),
+                           KEY `manufacurer_id_idx` (`manufacturer_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=16 DEFAULT CHARSET=utf8mb3;
+
+
+CREATE TABLE `cars` (
+                        `id` bigint NOT NULL AUTO_INCREMENT,
+                        `model` varchar(255) COLLATE utf8_hungarian_ci DEFAULT NULL,
+                        `is_deleted` tinyint DEFAULT '0',
+                        `manufacturer_id` bigint NOT NULL,
+                        PRIMARY KEY (`id`),
+                        KEY `manufacturer_id_idx` (`manufacturer_id`),
+                        CONSTRAINT `manufacturer_id` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`id`)
```

Comment:
У відео Богдан пояснював як ми маємо назвати CONSTRAINT
```suggestion
                        CONSTRAINT `cars_manufacturers_fk` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`id`)
```